webapp/src/widgets/menu/textOption.tsx

```diff
@@ -26,7 +27,7 @@ function TextOption(props:TextOptionProps): JSX.Element {
                 props.onClick(props.id)
             }}
         >
-            {icon ?? <div className='noicon'/>}
+            <div className={`${check ? 'menu-option__check' : 'false'}`}>{icon ?? <div className='noicon'/>}</div>
```

Comment:
I don't think className should be set if `check=false`, probably at least set to empty('') rather than false.